modules/gke/providers.tf

```diff
@@ -9,9 +9,9 @@ terraform {
     helm = {
       source = "hashicorp/helm"
     }
-    http = {
```

Comment:
Not used by default, so let's comment to avoid `terraform init` to resolve/download it?